static/src/javascripts/projects/common/modules/identity/wizard.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+/* eslint-disable no-underscore-dangle, react/sort-comp */
+
+import bean from 'bean';
```

Comment:
Would it be possible to get rid of the dependency, but using `addEventListener()`? We are trying to refactor more and more code, to use native functionality where possible and bean is one of these ...